packages/flutter_tools/test/commands.shard/permeable/build_bundle_test.dart

```diff
@@ -8,6 +8,9 @@
 // dependencies have been fixed.
 // https://github.com/flutter/flutter/issues/85160
 // Fails with "flutter test --test-randomize-ordering-seed=20210722"
+//
+// The mixture of MemoryFileSystem and LocalFileSystem and the global state of
+// Cache.flutterRoot are blocking randomizing these tests.
```

Comment:
The changes to this file make it MORE hermetic; however it's still depending on global state.